crates/energy/benches/nearby.rs

```diff
@@ -53,8 +53,8 @@ fn update_index(
 
 fn init_world_with_entities_moving(world: &mut World, num_entities: &NumPoints) {
     let mut index = EntityIndex::new();
-    let max_point_value = MAP_SIZE - MOVEMENT_RADIUS * 2.;
-
+    let max_point_value =
+        UNIT_SPACING * (<NumPoints as Into<usize>>::into(*num_entities) as f32).sqrt();
```

Comment:
`num_entities.into::<usize>()` should work as well.

Reply:
That is not how into works. Into is a function with no type parameter of a struct implementing `Into<T>` and returning `T`. I did give it a go anyway and it did not compile.